cluster-autoscaler/config/const.go

```diff
@@ -17,8 +17,15 @@ limitations under the License.
 package config
 
 const (
+	// Defaults.
+
 	// DefaultMaxClusterCores is the default maximum number of cores in the cluster.
 	DefaultMaxClusterCores = 5000 * 64
 	// DefaultMaxClusterMemory is the default maximum number of gigabytes of memory in cluster.
 	DefaultMaxClusterMemory = 5000 * 64 * 20
+
+	// Useful universal constants.
+
+	// Gigabyte is 2^30 bytes.
+	Gigabyte = 1024 * 1024 * 1024
```

Comment:
The fact this constant is in `config` module makes it look weird when it use used. Maybe consider creating separate module (consts/units?) for this one?

Reply:
I kind of agree, but I'd rather not add another module just for this constant

Reply:
What is expensive about modules?

Reply:
Complexity. I'll move it since this is cleaner, it just seems like a bit of an overkill

Reply:
Done

Reply:
Thanks. Can we rename `MB` to `Megabyte` and move it there too? Maybe there is some `KB` also? :)

Reply:
Do you mean MB from unit tests? If so, we don't use that in non-test code, so I'd rather keep it contained to test files

Reply:
Yeah - I meant those. But I did not notice those are test only. If so leave it that way :)